src/modules/currency/__tests__/formatNumber.test.ts

```diff
@@ -0,0 +1,126 @@
+import formatNumber from '../formatNumber';
+
+describe('formatNumber', () => {
+  it('should format the amount with default options', () => {
+    const result = formatNumber('1000.5', { currency: 'USD' });
+    expect(result).toBe('$1,000.50');
+  });
+
+  //   FAILING
```

Comment:
failing ?

Reply:
This test case is failing, not sure why. Expected and received values are the same.

Reply:
Discussed.